source/sphinx_extensions/command_block/extension.py

```diff
@@ -40,17 +63,34 @@ class CommandBlockDirective(docutils.parsers.rst.Directive):
     has_content = True
```

Comment:
I spoke too soon. Apparently this *is* important. I suspect Sphinx might make use of it in ways I do not understand. Ultimately, my findings are: it stays! :clinking_glasses: